README.md

```diff
@@ -1,45 +1,48 @@
 ## Opinionated Serverless Flask
-Comes with a lot of useful stuff ready to go.
 
+Comes with a lot of useful stuff ready to go.
 
 ## Features
-* [Flask Smorest](https://pypi.org/project/flask-smorest/)
-  * OpenAPI
-  * Swagger UI
-  * Redoc
-* [Pytest using database subtransactions](https://pypi.org/project/pytest-flask-sqlalchemy/) for test isolation
-* [Pytest-FactoryBoy](https://pytest-factoryboy.readthedocs.io/en/latest/#model-fixture) for generating sample data for seeding DB and tests.
-* Flake8 with [Mypy](http://mypy-lang.org/) for type-checking
-* AWS Lambda integration
-  * [Serverless AWS](https://serverless.com/framework/docs/providers/aws/)
-  * Load Flask config from AWS Secrets Manager
-* [Flask-Migrate](https://flask-migrate.readthedocs.io/en/latest/) for DB migrations with alembic and Flask CLI
-* [Pipenv](https://pipenv.readthedocs.io/en/latest/) for dependency management
 
+- [Flask Smorest](https://pypi.org/project/flask-smorest/)
+  - OpenAPI
+  - Swagger UI
+  - Redoc
+- [Pytest using database subtransactions](https://pypi.org/project/pytest-flask-sqlalchemy/) for test isolation
+- [Pytest-FactoryBoy](https://pytest-factoryboy.readthedocs.io/en/latest/#model-fixture) for generating sample data for seeding DB and tests.
+- Flake8 with [Mypy](http://mypy-lang.org/) for type-checking
+- AWS Lambda integration
+  - [Serverless AWS](https://serverless.com/framework/docs/providers/aws/)
+  - Load Flask config from AWS Secrets Manager
+- [Flask-Migrate](https://flask-migrate.readthedocs.io/en/latest/) for DB migrations with alembic and Flask CLI
+- [Poetry](https://python-poetry.org/docs/) for dependency management
 
 ## Quickstart:
 
 ### Prerequisites:
+
 ```
-brew install pipenv  # mac
-python -m pip install --upgrade pip pipenv  # generic
```

Comment:
Pipenv was used only for packaging so you replaced it with poetry? 
Shouldn't pipenv still be used for local venv management?

Reply:
poetry manages virtual environments the same way as pipenv. run `poetry shell` to enter your venv.

Reply:
Thanks.